doc/sphinx/src/models.rst

```diff
@@ -1491,16 +1490,82 @@ key-value pairs. For exampe the following input deck is for air:
   shrinklTBounds = 0.15
   shrinkleBounds = 0.5
 
-The only required value in an input file is the matid, in this
-case 5030. All other values will be inferred from the original sesame
-database if possible and if no value in the input file is
-provided. Comments are prefixed with ``#``.
+Comments are prefixed with ``#``. `eospac`_ uses environment variables
+and files to locate files in the `sesame`_ database, and
+``sesame2spiner`` uses `eospac`_. So the location of the ``sesame``
+database need not be provided by the command line. For how to specify
+`sesame`_ file locations, see the `eospac`_ manual.
 
-`eospac`_ uses environment variables and files to locate files in the
-`sesame`_ database, and ``sesame2spiner`` uses `eospac`_. So the
-location of the ``sesame`` database need not be provided by the
-command line. For how to specify `sesame`_ file locations, see the
-`eospac`_ manual.
+Piecewise Spiner Grids
+````````````````````````
+
+``SpinerEOS`` models and ``sesame2spiner`` also support grids with
+different resolutions in different parts of the table. We call these
+**piecewise** grids. This can be enabled or disabled with the
+parameters
+
+.. code-block::
+
+  piecewiseRho = true
+  piecewiseT = true
+  piecewiseSie = true
+
+These options may be true or false. The default is true. When
+piecewise grids are active, the density grid gets split into three
+pieces, a piece of diameter ``rhoFineDiameterDecades`` (in log space)
+around the material's normal density and a piece above and below. The
+``numrho/decade`` parameter sets the number of points per decade in
+this central refined region. The regions at lower and higher density
+have ``rhoCoarseFactorLo`` and ``rhoCoarseFactorHi`` fewer points per
+decade respectively compared to the finer region.
```

Comment:
> This whole description would benefit from an image or a diagram.

That's fair. Added.

> Also, is this the best way of inputting this? Why not something like a rhominfine->rhomaxfine? I know that the same effect can be specified by adjusting quantities specified below, and maybe this is the best way for it to be internally represented, but it might be reasonable to permit a different kind of input. Just a thought there.

I agree. I chose the model I chose here because it seemed reasonable to me in the moment. Obviously `rhominfine/rhomaxfine` is a bit more flexible. I added the ability to set `rhoMinFine` and `rhoMaxFine`.